OIPA/api/sector/views.py

```diff
@@ -7,17 +7,75 @@
 
 
 class SectorList(ListAPIView):
+    """
+    Returns a list of IATI Sectors stored in OIPA.
+
+    ## Request parameters
+
+    - `aggregations` (*optional*): Aggregate available information.
```

Comment:
aggregations work differently in views that are not an ActivityList view. AggregationSerializer is a field in SectorSerializer. Aggregation is performed on each separate sector. To aggregate the total_budget of all activities in each sector: `/api/sectors?fields[aggregations]=total_budget`